RELEASE.md

```diff
@@ -99,3 +99,46 @@ Perform the following actions ONLY after the vote is successful and closed.
    svn add {version}
    svn commit -m “Apache Ignite {version}”
    ```
+
+
+## Gradle commands
+1. Fill sign information in gradle.properties file.
```

Comment:
I think it would be nice to mention that all these properties should be managed through the gradle.properties file in the user’s Gradle home directory instead of using this file directly in order to avoiding accidental commit private info.